src/tools/clippy/tests/ui/unknown_clippy_lints.stderr

```diff
@@ -1,52 +1,58 @@
-error: unknown clippy lint: clippy::if_not_els
+error: unknown lint: `clippy::All`
+  --> $DIR/unknown_clippy_lints.rs:5:10
+   |
+LL | #![allow(clippy::All)]
+   |          ^^^^^^^^^^^ help: did you mean: `clippy::all`
+   |
+   = note: `-D unknown-lints` implied by `-D warnings`
```

Comment:
Hmm, I'm curious to know why this changed. This is a bug fix, right, it should have given a warning before?

Reply:
Hm? The order is just different. And it has the default weirdness with linting attributes, that it is duplicated for the first inner attribute.

Reply:
Ah ok, I misread. No need to change anything.

> And it has the default weirdness with linting attributes, that it is duplicated for the first inner attribute.

Does this have an issue open? It seems worth fixing.

Reply:
I don't know of an issue for this.

Reply:
I opened https://github.com/rust-lang/rust-clippy/issues/6602.

Reply:
I'm not sure if it's just in a different order; at the bottom it shows that it emitted 8 errors previously and now it emits 9.